lib/test_helpers/pdf_comparison_test_factory.rb

```diff
@@ -106,14 +110,15 @@ def self.create_line_items_request(distribution, partner, storage_creation)
 
   def self.create_dist(partner, storage_creation, delivery_method)
     Time.zone = "America/Los_Angeles"
+    travel_to(Time.zone.local(2024, 12, 30, 0o0, 0o0, 0o0))
```

Comment:
If you're going to use this within a method, you should use the block approach so that it doesn't affect the rest of the calling code. I.e. `travel_to(...) do ... end`.

Also, what is `0o0`? Is there a reason plain `0` doesn't work?

Reply:
resolved in [84564e9](https://github.com/rubyforgood/human-essentials/pull/4540/commits/84564e9a000e28af5c136a025cd5ce52eade02f9)

0o0 is an octal zero, not sure what happened but I changed it to normal 0